Nybble/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+  "configuration": {
+    "title": "NybbleConfiguration",
+    "type": "object",
+    "properties": {
+      "nhub_url": {
+        "title": "Nhub Url",
+        "description": "Nybble Hub Connector Base URL",
+        "minLength": 1,
+        "maxLength": 2083,
+        "format": "uri",
+        "type": "string"
+      },
+      "nhub_username": {
+        "title": "Nhub Username",
+        "description": "Nybble Hub Connector username",
+        "type": "string"
+      },
+      "nhub_key": {
+        "title": "Nhub Key",
+        "description": "Nybble Hub Connector Key to authenticate the requests",
+        "type": "string"
+      },
+      "sekoia_url": {
+        "title": "Sekoia Url",
+        "description": "Sekoia.io Base URL",
+        "minLength": 1,
+        "maxLength": 2083,
+        "format": "uri",
+        "type": "string"
+      },
+      "sekoia_api_key": {
+        "title": "Sekoia Api Key",
+        "description": "Sekoia.io API Key",
+        "type": "string"
+      }
```

Comment:
I suggest removing these properties, as it's the Sekoia.io automation module that must provide the ability to get the definition of a rule.